pages/linux/qm-list.md

```diff
@@ -0,0 +1,12 @@
+# qm list
+
+> List all virtual machines.
+> More information: <https://pve.proxmox.com/pve-docs/qm.1.html>.
+
+- List all virtual machines:
+
+`qm list`
+
+- List full status all virtual machines:
+
+`qm list --full 1`
```

Comment:
```suggestion
`qm list --full {{boolean}}`
```
Or
```suggestion
`qm list --full {{1}}`
```
As it will tell the reader that this is a placeholder.